collectors/build/jenkins-audit/src/main/java/com/capitalone/dashboard/collector/AuditCollector.java

```diff
@@ -0,0 +1,37 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.model.Collector;
+import com.capitalone.dashboard.model.CollectorType;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AuditCollector extends Collector {
+
+    public static final String INSTANCE_URL = "instanceUrl";
+    public static final String JOB_NAME = "jobName";
+    public static final String JOB_URL = "jobUrl";
+    private List<String> buildServers = new ArrayList<>();
+
+    public static AuditCollector prototype(List<String> servers) {
```

Comment:
please add method descriptions. This applies for all new methods you have added

Reply:
method description added wherever nacessary